src/sentry/tasks/store.py

```diff
@@ -403,9 +403,11 @@ def save_attachment(event, attachment):
     # store_crash_reports setting. Otherwise, we assume that the client has
     # already verified PII and just store the attachment.
     if attachment.type in CRASH_REPORT_TYPES:
-        if not event.project.get_option(
+        project = Project.objects.get_from_cache(id=event.project_id)
+        organization = Organization.objects.get_from_cache(id=project.organization_id)
+        if not project.get_option(
             "sentry:store_crash_reports"
-        ) and not event.project.organization.get_option("sentry:store_crash_reports"):
+        ) and not organization.get_option("sentry:store_crash_reports"):
```

Comment:
Th eonly suggestion I'd have is to make this slightly more efficient and split these up:

```python
project = Project.objects.get_from_cache(id=event.project_id)
if not project.get_option("sentry:store_crash_report"):
    organization = Organization.objects.get_from_cache(id=project.organization_id)
    if not organization.get_option("sentry:store_crash_reports"):
        return
```

Since these are network calls, it's worth splitting up the if statements so it only does the second call if needed.